src/Types.hs

```diff
@@ -359,8 +359,8 @@ instance SOP.Generic ProtoDelegation
 
 data DelegationContext =
     DelCtxIdeaSpace { _delCtxIdeaSpace :: IdeaSpace  }
-  | DelCtxTopic     { _delCtxTopicId   :: AUID Topic }
-  | DelCtxIdea      { _delCtxIdeaId    :: AUID Idea  }
+  | DelCtxTopicId   { _delCtxTopicId   :: AUID Topic }
```

Comment:
Nitpick we should rename `DelCtx...` to `DlgCtx...`. First I thought this is about deletion of some context.